src/components/FruitDisplays/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react"
+import "src/styles/tailwind.css"
+import { Segmented, Space, Button } from "antd"
+import { CloseOutlined, EditOutlined } from "@ant-design/icons"
+
+function FruitDisplays({ id, names, prices, onDelete, onEditButtonClick }) {
+  return (
+    <div style={{ "margin-right": "4%" }}>
+      <div
+        style={{
+          "margin-right": "2%",
+          "margin-left": "2%",
+        }}
+      >
+        <div style={{ "margin-bottom": "3em", "margin-top": "2em" }} />
+        <Space direction="vertical">
+          <Segmented
+            options={[
+              {
+                label: (
+                  <div
+                    style={{
+                      "padding-top": ".25em",
+                      "padding-right": ".5em",
+                      "padding-left": ".5em",
+                    }}
+                  >
+                    <img
+                      src={require("./exampleimage.png")}
+                      style={{ "border-radius": "30px" }}
+                    ></img>
```

Comment:
Some tags or when you don't need to wrap any children. Use more self-closing tag. Example: <img ... />